parsec/core/gui/qrcode_widget.py

```diff
@@ -21,21 +21,6 @@
 
 
 def generate_qr_code(text):
-    global PARSEC_LOGO
-
-    if not PARSEC_LOGO:
-        img = QImage(":/logos/images/logos/parsec2.png")
```

Comment:
`:/logos/images/logos/parsec2.png` doesn't seem to be used anywhere else in the code base, so shouldn't this file be removed as well ?

Reply:
Yep, it has been removed.